lookout/style/typos/utils.py

```diff
@@ -32,42 +32,51 @@ def read_frequencies(file: str) -> Dict[str, int]:
     """
     Read token frequencies from the file.
 
-    :param file: Path to the file containing tokens with frequencies.
-                 Every line has format "token count".
+    :param file: Path to the file containing space-separated word-frequency pairs one-per-line. \
+                 File should be in .csv or compressed .csv.xz format.
     :return: Dictionary of tokens frequencies.
     """
     frequencies = {}
-    with lzma.open(file, "rt") as f:
-        for line in f:
-            split = line.split()
-            frequencies[split[0]] = int(split[1])
+    try:
```

Comment:
Let's use https://github.com/RaRe-Technologies/smart_open

Reply:
I will submit xz support there, let's use bzip2 for now

Reply:
This .xz support will be removed completely in the upcoming PRs. I still should change this?

Reply:
Then cool, use this package as-is.

FWIW https://github.com/RaRe-Technologies/smart_open/pull/260

Reply:
As-is it just doesn't work. Or I don't understand what you mean by "as-is", or I will have to change formats of several files from tests, and then change them back in the upcoming PRs.